tests/unit/iotics/lib/identity/conftest.py

```diff
@@ -83,9 +83,14 @@ def other_key_pair(other_key_pair_secrets):
 
 @pytest.fixture
 def doc_did():
-    return 'did:iotics:iotHHHHKpPGWyEC4FFo4d6oyzVVk6MXLmEgY'
+    return 'did:iotics:iotDadb3rSWedk8iqExSbwqLtijG5XQByHC7'
```

Comment:
since this is a magic string that's repeated in many tests, would it be better to have it in a constant with a meaningful name

Reply:
Yes it would be clearer. I did not want to refactor too much of the tests. I had to override this value because it does not pass the identifier checksum validation.